packages/backend-modules/republik-crowdfundings/lib/payments/Pledge.js

```diff
@@ -75,6 +76,7 @@ const afterChange = async ({ pledge }, context) => {
     pledge.status === 'SUCCESSFUL' && refreshPotForPledgeId(pledge.id, context),
     pledge.status === 'PAID_INVESTIGATE' &&
       slack.publishPledge(user, pledge, 'PAID_INVESTIGATE'),
+    rewardSender(pledge, context),
   ]).catch((e) => {
     console.error('error in afterChange', e)
```

Comment:
Would it make sense to separate the promises for the pledge confirmation and the rewardSender? If something goes wrong for the sender reward, we don't want to have an error for the user who bought the subscription, I guess?

Reply:
It is my understanding that `Promise.all` will fail immediatly, but pending Promises be fullfilled after all; if `rewardSender` fails (aka. is rejected), `sendPledgeConfirmation` won't be aborted.